src/xdocs/config_imports.xml

```diff
@@ -603,11 +603,26 @@ import android.*;
           </tr>
           <tr>
             <td>illegalPkgs</td>
-            <td>packages to reject</td>
+            <td>Packages to reject, by default checks if import is the part of package. If <b>regexp</b> variable is set, then list of packages will be
+                interpreted as regular expressions. Note, all properties for match will be used. Order of matching: <b>illegalPkgs</b>, <b>illegalClasses</b></td>
             <td><a href="property_types.html#stringSet">String Set</a></td>
             <td><code>sun</code></td>
           </tr>
+          <tr>
+            <td>illegalClasses</td>
+            <td>Class names to reject, by default checks if import equals class name. If <b>regexp</b> variable is set, then list of class name will be
+                interpreted as regular expressions. Note, all properties for match will be used. Order of matching: <b>illegalPkgs</b>, <b>illegalClasses</b></td>
+            <td><a href="property_types.html#stringSet">String Set</a></td>
+            <td><code>{}</code></td>
+          </tr>
+          <tr>
+            <td>regexp</td>
+            <td>Whether the packages and class names should be interpreted as regular expressions</td>
```

Comment:
change `packages and class names` to their property names. It is unclear you are talking about them.